server/tests/server-test/tests/idlestate_test.cpp

```diff
@@ -88,4 +92,344 @@ TEST(IdleStateTest, update_possible_actions)
     game.stateChange<IdleState>();
 
     EXPECT_EQ(2, request_spy.size());
-}
\ No newline at end of file
+}
+
+TEST(IdleStateTest, buy_house)
+{
+    Game game(fieldsTwoGroups());
+    game.players() = RingBuffer<Player>(std::vector<Player>{ { player_1 } });
+    game.stateChange<IdleState>();
+    game.bank().createAccount(player_1, 1000);
+
+    dynamic_cast<Street*>(game.board()[0])->changeOwner(player_1);
+    dynamic_cast<Street*>(game.board()[1])->changeOwner(player_1);
+
+	QSignalSpy property_spy(&game, &Game::onPropertyChange);
+	QSignalSpy money_spy(&game, &Game::onMoneyChange);
+
+    game.requestBuyHouse(player_1, 0, true);
+
+	EXPECT_EQ(1, property_spy.size());
```

Comment:
Allignment.

Reply:
Copy-paste error, will un-tabify the whole file/folder/project(?)